linker/shared/src/test/scala/org/scalajs/linker/LibrarySizeTest.scala

```diff
@@ -70,9 +70,9 @@ class LibrarySizeTest {
     )
 
     testLinkedSizes(
-      expectedFastLinkSize = 147046,
-      expectedFullLinkSizeWithoutClosure = 85355,
-      expectedFullLinkSizeWithClosure = 21492,
+      expectedFastLinkSize = 148624,
```

Comment:
Just to check my understanding: This is because of (signed) integer division now being inlined more strongly? (IMO OK).

Reply:
Ah, I see. OK, that's acceptable. Maybe an alternative could be to emit runtime long methods as static methods, so it would be easier for JS optimizers to get rid of them. (it's not entirely clear to me how easy that would be in practice :-/).